config/initializers/1_redirect_stdout.rb

```diff
@@ -0,0 +1,6 @@
+# In Passenger environment standard output and error is not logged at all.
+# Using this technique it is possible to redirect all the output to standard
+# Rails log.
+
+$stderr.reopen Rails.logger.instance_variable_get(:@logdev).dev
```

Comment:
Tests are failing here:

<pre>
00:03:40.685 rake aborted!
00:03:40.685 NoMethodError: undefined method `dev' for nil:NilClass
00:03:40.686 /var/lib/workspace/workspace/test_develop_pr_core@2/database/postgresql/ruby/1.9.3/config/initializers/1_redirect_stdout.rb:5:in `<top (required)>'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/activesupport-3.2.21/lib/active_support/dependencies.rb:245:in `load'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/activesupport-3.2.21/lib/active_support/dependencies.rb:245:in `block in load'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/activesupport-3.2.21/lib/active_support/dependencies.rb:236:in `load_dependency'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/activesupport-3.2.21/lib/active_support/dependencies.rb:245:in `load'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/engine.rb:593:in `block (2 levels) in <class:Engine>'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/engine.rb:592:in `each'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/engine.rb:592:in `block in <class:Engine>'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/initializable.rb:30:in `instance_exec'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/initializable.rb:30:in `run'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/initializable.rb:55:in `block in run_initializers'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/initializable.rb:54:in `each'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/initializable.rb:54:in `run_initializers'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/application.rb:136:in `initialize!'
00:03:40.686 /usr/local/rvm/gems/ruby-1.9.3-p392@test_develop_pr_core-2/gems/railties-3.2.21/lib/rails/railtie/configurable.rb:30:in `method_missing'
00:03:40.686 /var/lib/workspace/workspace/test_develop_pr_core@2/database/postgresql/ruby/1.9.3/config/environment.rb:12:in `<top (required)>'
</pre>